docs/UserGuide.md

```diff
@@ -114,20 +119,23 @@ GME data is saved as a JSON file `[JAR file location]/data/GMEdata.json`. Advanc
 
 Adds a new passenger in the GME terminal.
 
-**Format:** `add n/NAME p/PHONE a/ADDRESS d/TRIPDAY t/TRIPTIME [tag/TAG tag/TAG ...]`
+**Format:** `add n/NAME p/PHONE a/ADDRESS d/TRIPDAY t/TRIPTIME [pr/PRICE] [tag/TAG tag/TAG ...]`
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the add command:**<br>
 * [TripDay](#6-glossary) is required to be a valid day of the week. e.g. `SUNDAY` or `FRIDAY`
 * [TripTime](#6-glossary) is required to be in the 24-hour format. e.g. `0530` or `2359` 
+* [Tag](#6-glossary) is required to be an alphanumeric input at most 30 characters in length.
```

Comment:
in the Tag model inside the code, i have put it as "at a maximum of 30 characters." refer to model if you feel like we should standardize.

Reply:
Also, perhaps you could add the restrictions for tags in the Edit Command section